### PR TITLE
Get Eldritch Attunement During Dailies (On GSH)

### PR DESCRIPTION
**Pull request by Tokoeka**

### CODE REVIEW

**Comment by Tokoeka** [comment]
Currently just equips water wings, as I don't think we care about the stat buff

**Comment by Tokoeka** [comment]
Possibly needs some form of "only get if you have free fights to use it on" preq, not sure if we have a "free fight sources" counter

**Comment by horrible-little-slime** [comment]
Is this tested? I know we generally have Attunement support, but I want to make sure that's been verified to still work post-paste.

File: src/dailies.ts [hunk 1]
```diff
@@ -666,3 +671,11 @@ function comb(): void {
   const combs = 11 - get("_freeBeachWalksUsed");
   cliExecute(`combo ${combs}`);
 }
+
+function getAttuned(): void {
+  if (holiday() === "Generic Summer Holiday" && !haveEffect($effect`Eldritch Attunement`)) {
```
Comment (horrible-little-slime) [inline src/dailies.ts:676]: use have(), please

File: src/fights.ts [hunk 2]
```diff
@@ -529,11 +530,18 @@ function bestFairy() {
 class FreeFight {
   available: () => number | boolean;
   run: () => void;
+  tentacle: () => boolean;
```
Comment (horrible-little-slime) [inline src/fights.ts:533]: I don't see the utility in making this a function rather than a boolean--it seems like a property that does not change.

**Review by horrible-little-slime — CHANGES_REQUESTED** [review]

File: src/fights.ts (hunk 2 above)
Comment (Tokoeka) [reply src/fights.ts:533]: Yeah, I am still a bit hazy on when to use one or tother, so I was just following how available and run were done. Changing

**Pull request merged**